Google.PowerShell.IntegrationTests/BigQuery/BigQuery.GcbqTable.Tests.ps1

```diff
@@ -0,0 +1,58 @@
+﻿. $PSScriptRoot\..\BigQuery\GcbqCmdlets.ps1
+$project, $zone, $oldActiveConfig, $configName = Set-GCloudConfig
+
+Describe "Get-GcbqTable" {
+
+    It "should list tables from a dataset object from pipeline."{
+        $tables = Get-GcbqDataset “pshell_testing” | Get-GcbqTable
```

Comment:
As I mentioned above as well, I was planning to iterate on the tests as I developed stuff like New-GcbqTable and Remove-GcbqTable.  Ideally by the end of this block of cmdlets, I would like to be using all of them anyway instead of external cmds like "bq".  That way the tests will eventually be contained individually and show that the cmdlets work properly together.